NuRadioReco/framework/station.py

```diff
@@ -92,9 +92,35 @@ def iter_channels(self, use_channels=None, sorted=False):
         for channel_id in channel_ids:
             yield self.get_channel(channel_id)
 
+    def iter_trigger_channels(self, use_channels=None):
+        for channel_id, channel in iteritems(self.__channels):
+            if use_channels is None:
+                yield channel.get_trigger_channel()
+            else:
+                if channel_id in use_channels:
+                    yield channel.get_trigger_channel()
+
     def get_channel(self, channel_id):
         return self.__channels[channel_id]
 
+    def get_trigger_channel(self, channel_id):
```

Comment:
we could also not implement this function to make clear that `trigger_channels` are not saved at station level, but are a sub-element of a channel. Users would need to write `station.get_channel(id).get_trigger_channel()` instead of `station.get_trigger_channel(id)`

Reply:
I added this interface for connivence because it makes the diff necessary to the trigger module minimal (I have not applied it to any but the hi-lo one yet). We could delete it for the reason you mention..